### PR TITLE
SB-1021: Cron Jobs logging

### PR DESCRIPTION
**Pull request by sbespalov**
Every cron job will put logs into separate log file named by Cron Job class name.

### CODE REVIEW

File: strongbox-parent/pom.xml [hunk 1]
```diff
@@ -660,6 +660,12 @@
                 <version>${version.logback}</version>
             </dependency>
 
+            <dependency>
+                <groupId>org.codehaus.janino</groupId>
+                <artifactId>janino</artifactId>
```
Comment (carlspring) [inline strongbox-parent/pom.xml]: What is this dependency for and under what license is it?

File: strongbox-resources/strongbox-common-resources/src/main/resources/logback.xml [hunk 2]
```diff
@@ -8,28 +8,51 @@
...
-        <rollingPolicy class="ch.qos.logback.core.rolling.SizeAndTimeBasedRollingPolicy">
-            <fileNamePattern>strongbox_%d{yyyy-MM-dd}.%i.log</fileNamePattern>
+    <appender name="FILE" class="ch.qos.logback.classic.sift.SiftingAppender">
 
-            <fileNamePattern>strongbox-%d{yyyy-MM-dd}.%i.log</fileNamePattern>
-            <maxFileSize>128MB</maxFileSize>
-            <maxHistory>31</maxHistory>
-            <totalSizeCap>1GB</totalSizeCap>
+        <discriminator
+            class="ch.qos.logback.classic.sift.MDCBasedDiscriminator">
```
Comment (carlspring) [inline strongbox-resources/strongbox-common-resources/src/main/resources/logback.xml]: Would you mind enlightening me what this does?

**Comment by carlspring** [comment]
@steve-todorov , @fuss86 : FYI.

**Comment by carlspring** [comment]
Thanks, @sbespalov!

File: strongbox-resources/strongbox-common-resources/src/main/resources/logback.xml (hunk 2 above)
Comment (sbespalov) [reply strongbox-resources/strongbox-common-resources/src/main/resources/logback.xml]: this produce log files based on Discriminator value, which is stored in ThreadLocal

File: strongbox-parent/pom.xml (hunk 1 above)
Comment (sbespalov) [reply strongbox-parent/pom.xml]: it's under BSD

https://en.wikipedia.org/wiki/BSD_licenses

Comment (sbespalov) [reply strongbox-parent/pom.xml]: used [here](https://github.com/strongbox/strongbox/pull/565/files#diff-846a8c163fa25f137004008783538413R14)

File: strongbox-resources/strongbox-common-resources/src/main/resources/logback.xml (hunk 2 above)
Comment (sbespalov) [reply strongbox-resources/strongbox-common-resources/src/main/resources/logback.xml]: this explained [here](https://logback.qos.ch/manual/appenders.html#SiftingAppender)

**Pull request merged**